server/datastore/mysql/teams.go

```diff
@@ -244,15 +253,17 @@ WHERE
 // fleet.ListOptions
 func (ds *Datastore) ListTeams(ctx context.Context, filter fleet.TeamFilter, opt fleet.ListOptions) ([]*fleet.Team, error) {
 	query := fmt.Sprintf(`
-			SELECT *,
+			SELECT `+teamColumns+`,
 				(SELECT count(*) FROM user_teams WHERE team_id = t.id) AS user_count,
 				(SELECT count(*) FROM hosts WHERE team_id = t.id) AS host_count
 			FROM teams t
 			WHERE %s
 		`,
 		ds.whereFilterTeams(filter, "t"),
 	)
-	query, params := searchLike(query, nil, opt.MatchQuery, teamSearchColumns...)
+	// We must normalize the name for full Unicode support (Unicode equivalence).
+	matchQuery := norm.NFC.String(opt.MatchQuery)
```

Comment:
Why would `opt.MatchQuery` have unicode? Does it include the team name?

Reply:
This is the `query` API param coming from the user. So it can contain unicode.
https://fleetdm.com/docs/rest-api/rest-api#list-teams